schema/spacewalk/upgrade/susemanager-schema-4.3.12-to-susemanager-schema-4.3.13/001-add-gpgkey-to-suseProductSCCRepository.sql

```diff
@@ -1,3 +1,13 @@
-ALTER TABLE suseProductSCCRepository ADD COLUMN IF NOT EXISTS gpg_key_url  VARCHAR(256);
-ALTER TABLE suseProductSCCRepository ADD COLUMN IF NOT EXISTS gpg_key_id   VARCHAR(14);
-ALTER TABLE suseProductSCCRepository ADD COLUMN IF NOT EXISTS gpg_key_fp   VARCHAR(50);
+DO $$
```

Comment:
@mcalmer why do we need to change this update script? It's a old one and they are running iteratively, so if a user is on older suma, when running this script the table must exist, if the user is in a newer schema version then this script would not run. 
What I'm missing in here?

Reply:
Because i renamed the table and we have the idempotent test bailing out.
We test from 4.3.8 migration to the latest 5.1.X version and we do it 2 times.
The first time the table still has the name and later the name is changed.
When it run a second time, there is no table with the name "suseProductSCCRepository" and this migration would fail.
So we need to check if a table exists with this name before we try to execute the Alter table statements.

And this PR was backported also to 4.3 branch

Reply:
got it. Thank you for the insides